corehq/apps/userreports/migrations/0001_add_inserted_at_timestamp.py

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+import logging
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.db.utils import DatabaseError
+
+from dimagi.utils.couch import sync_docs
+from corehq.apps.userreports import models as userreports_models
+from corehq.apps.userreports.sql import get_table_name
+
+logger = logging.getLogger(__name__)
+
+
+class Migration(DataMigration):
```

Comment:
`DataSourceConfiguration` is a couch model, so I'm pretty sure most of this is unnecessary?